frontend/src/modules/labworkSteps/actions.ts

```diff
@@ -47,7 +47,16 @@ export function initSamplesAtStep(stepID: FMSId) {
       const templateActions = selectSampleNextStepTemplateActions(getState())
 
       // Request the list of templates for the protocol
-      const templatesResponse = await dispatch(api.sampleNextStep.prefill.templates(protocol.id))
+      const templatesResponse = await dispatch(api.sampleNextStep.prefill.templates(protocol.id)).then(
+        (response) => {
+          let filteredResponse = { ...response }
+          if (!step.needs_planning){
+            filteredResponse.data = response.data.filter((template) => {return !template.description.includes("planning")})
+          }
+          return response
+        }
+      )
+
```

Comment:
Um... what's the user of `filteredResponse`?

Reply:
I believe you can just use `response.data` directly

Reply:
err looks like i pushed the wrong version ... filteredresponse is supposed to be returned. Not sure how this ended up being pushed. Let me fix that... response.data would not fix the issue...